src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos/AspireAzureEFCoreCosmosDBExtensions.cs

```diff
@@ -141,4 +106,72 @@ void UseCosmosBody(CosmosDbContextOptionsBuilder builder)
             }
         }
     }
+
+    /// <summary>
+    /// Configures retries, health check, logging and telemetry for the <see cref="DbContext" />.
```

Comment:
```suggestion
    /// Configures logging and telemetry for the <see cref="DbContext" />.
```

There's no retry or health checks being configured.